src/Dnt.Commands/Packages/SwitchDllsToProjectsCommand.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Dnt.Commands.Infrastructure;
+using Dnt.Commands.Packages.Switcher;
+using Microsoft.Build.Construction;
+using Microsoft.Build.Evaluation;
+using NConsole;
+
+namespace Dnt.Commands.Packages
+{
+    [Command(Name = "switch-dll-references-to-projects")]
+    public class SwitchDllReferencesToProjectsCommand : CommandBase
+    {
+        [Argument(Position = 1)]
+        public string Solution { get; set; }
+
+        public override async Task<object> RunAsync(CommandLineProcessor processor, IConsoleHost host)
+        {
+            var projects = GetProjects(Solution, host);
+
+            ReplaceDllReferencesWithProjects(projects, host);
+
+            return null;
+        }
+
+        private IEnumerable<ProjectInSolution> GetProjects(string solutionPath, IConsoleHost host)
```

Comment:
Please use GetProjectPaths here so that the logic is the same as in the other commands: https://github.com/RSuter/DNT/blob/master/src/Dnt.Commands/ProjectCommandBase.cs#L14

Then load the project from the path into a single collection, like here: 
https://github.com/RSuter/DNT/blob/master/src/Dnt.Commands/Projects/EnableCommand.cs#L22

Reply:
Good points, updated :)
I made a change to GetProjectPaths to exclude solution folders, but I could achieve the effect I need in the command instead if you prefer.
Also added command description and switched it to the projects namespace